### DISCUSSION TITLE
RLC-520A_v2 FW 3.1.0.2368_23062701

### DISCUSSION THREAD

**Discussion by KultusX** [AT0myks/reolink-fw-archive#25] [Firmware sharing]
Hi guys, I found here on github many good ideas and help. So I want to pay back and here is the link for newest FW for this favourite model. HW ver. IPC_523128M5MP_V2:

https://support-d.reolink.com/attachments/token/t4dIPW2Qw4zgWGliwvDCbFSGK/?name=IPC_523128M5MP_V2.2368_23062701.RLC-520A.OV05A10.5MP.REOLINK.pak

**Comment by jcsouthworth** [comment]
I installed this on two RLC-510A (IPC_523128M5MP_V2) and it appears to work well. The devices now report they are RLC-520A. The install was a little clunky in that I needed to clear the cookies from my browser before I could login - something I hadn't needed to do with other versions for the RLC-510A. This release adds pet detection.

**Reply by coltc50** [reply]
Thank you... Did the same and was surprised that it works. Now 510 has Pet detection...

**Reply by AT0myks** [reply]
@crycode-de shared this firmware for the RLC-510A V2 in #41.

**Comment by AT0myks** [comment]
Thanks for the contribution, added in 3ec625a6d1a6e413d5ef23f5977799bb3e6942cc.

**Comment by milandzuris** [comment]
Work this Firmware on classic 520A? V1?

@AT0myks @KultusX thx

**Reply by AT0myks** [reply]
I can't say for sure that this firmware is incompatible with the V1, but even if it is your camera will probably give you an error if you try it. You can ask Reolink support if they have an update for the V1.
